app/invocation/invocation_model.tsx

```diff
@@ -249,6 +250,15 @@ export default class InvocationModel {
     return this.invocations.find(() => true)?.acl?.groupId === "";
   }
 
+  isReadOnlyInvocation(): boolean {
```

Comment:
nit: consider renaming to something like `usedReadOnlyAPIKey()` or maybe more directly, `hasCacheWriteCapability()` and negate the return & callsites

Reply:
Done